### PR TITLE
Images for documentation

### PR DESCRIPTION
**Pull request by MarPetra**
getting images from using GUI

### CODE REVIEW

File: pgRoutingLayer.py [hunk 1]
```diff
@@ -43,20 +43,16 @@
 class PgRoutingLayer:
 
     SUPPORTED_FUNCTIONS = [
-        'pgr_dijkstra',
+
```
Comment (cayetanobv) [inline pgRoutingLayer.py]: Remove empty line

File: pgRoutingLayer.py [hunk 2]
```diff
@@ -43,20 +43,16 @@
...
+        'pgr_bdAstar',
         'pgr_bdAstarCost',
-        # 'tsp_euclid',
-        # 'with_Points',
-        # 'with_PointsCost'
+        'pgr_bdDijkstra',
+        'pgr_bdDijkstraCost',
+        'pgr_dijkstra',
+        'pgr_dijkstraCost',
+        'pgr_KSP',
```
Comment (cayetanobv) [inline pgRoutingLayer.py]: Remove comma after 'pgr_KSP'

**Comment by cvvergara** [comment]
@cayetanobv 
Looks like @MarPetra has finished working on the documentation.
Can you re-review your review

File: builddoc.sh [hunk 3]
```diff
@@ -1,4 +1,4 @@
 #!/bin/bash
 rm -fR site
 mkdocs gh-deploy --clean
-rm -fR site
+#rm -fR site
```
Comment (cayetanobv) [inline builddoc.sh]: Why did you comment this line?

File: pb_tool.cfg [hunk 4]
```diff
@@ -6,7 +6,8 @@ name: pgRoutingLayer
 # Full path to where you want your plugin directory copied. If empty,
 # the QGIS default path will be used. Don't include the plugin name in
 # the path.
-plugin_path: ~/.local/share/QGIS/QGIS3/profiles/pgrouting/python/plugins
+# plugin_path: ~/.local/share/QGIS/QGIS3/profiles/pgrouting/python/plugins/
+plugin_path: /home/vicky/.local/share/QGIS/QGIS3/profiles/pgrouting/python/plugins
```
Comment (cayetanobv) [inline pb_tool.cfg]: We can not set this parameter with this absolute path in the repository. If you have problems building code you could use pb_tool in this way (with -p you can set your plugin path):
```
pb_tool deploy -y -p ~/.local/share/QGIS/QGIS3/profiles/default/python/plugins
```

**Pull request merged**